api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -93,9 +93,19 @@ message ClusterStats {
   //     sum_locality(total_error_requests) + total_dropped_requests`
   //
   // The total number of dropped requests. This covers requests
-  // deliberately dropped by the drop_overload policy and circuit breaking.
+  // deliberately dropped by the circuit breaking.
   uint64 total_dropped_requests = 3;
```

Comment:
I think this can also be deprecated, since it can be inferred from `dropped_requests` now. Better to be concise in APIs.

Reply:
Yes, I did not want to do this as it is implemented.
I am open to doing those changes in subsequent PRs